packages/components/pie-tag/src/tag.scss

```diff
@@ -2,12 +2,13 @@
 
 // Base tag styles
 .c-tag {
-    // The following values set to a default tag (neutral large)
-    --tag-bg-color: var(--dt-color-container-inverse);
-    --tag-color: var(--dt-color-content-inverse);
-    --tag-padding: 2px var(--dt-spacing-b);
-    --tag-height: 24px;
+    --tag-bg-color: var(--dt-color-container-strong);
+    --tag-color: var(--dt-color-content-default);
     --tag-border-radius: var(--dt-radius-rounded-b);
+    --tag-border-width: 1px;
+    --tag-border-color: transparent;
```

Comment:
Will the calculation of the padding that subtracts the border mean that visually the tag will look 2px less because of the transparent border?

Reply:
I believe no, the transparent border will inherit the background color allowing it to look the same.